Examples/LambdaFunctions/Package.swift

```diff
@@ -42,5 +50,12 @@ let package = Package(
         .target(name: "CurrencyExchange", dependencies: [
             .product(name: "AWSLambdaRuntime", package: "swift-aws-lambda-runtime"),
         ]),
+        .target(name: "LambdaAPI", dependencies: [
```

Comment:
its great to have such a comprehensive example! I think it needs a more specific name though - one that describes better the kind of things it demonstrates (see CurrencyExchange for example)

Reply:
I agree, what do you think about `ProductAPI` ?

Reply:
46ba794